docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM php:7.4-apache
+
+ARG BRANCH=stable
+
+COPY --chmod=755 ./entrypoint.sh /usr/local/sbin/entrypoint
```

Comment:
It's best to rename not to be so generic name, so in case you want/need to wrap and execute another one, like you do with the php entrypoint:

```sh
exec docker-php-entrypoint apache2-foreground
```

if the parent image also had it named as `entrypoint` you would need to make workarounds.

Hence, I propose to use a file named `dokuwiki-entrypoint` when installing (doesn't matter how you name it in your git repository).